### ISSUE TITLE
Looking for the config of the best results in NTU-interaction 

### ISSUE THREAD

**Issue by 519542630** [mgiant/2P-GCN#1]
question 1 : I ran your model and found that the effect on the NTU-Interaction dataset was much lower (3% lower) than shown in the article.  I have tried to adjust some of the parameter options, but my best results are still low.  Can you publish the best parameters and possible training steps? 
question 2 :  I have run CTR-GCN and 2s-AGCN before, but the results are much lower than what you mentioned in your article. My approach to NTU-Interaction is very simple, and I simply remove the single-person action category and keep the two-person action category. I would like to ask what about your data preprocessing method?

**Comment by mgiant** [comment]
Hi, thank you for your interest in our work. 
1. The config file contains all configurations needed to reproduce our results. There could be several reasons why you're seeing a lower performance on the NTU-Interaction dataset, such as data not being processed correctly, different versions of dependencies, etc. I have provided the pre-trained models, you can evaluate on NTU-Interaction with the corresponding config file and preprocessing steps as in `README.md`. If there is any problem, please let me know.
2. As for CTR-GCN and 2s-AGCN, we followed their respective data preprocessing and ensemble techniques to produce their results on NTU-Interaction. The only difference is we kept only the two-person action categories (50-60, 106-120) and mapped them to 0-index action labels.

Feel free to ask if you have further questions.
